examples/decomposition/plot_image_denoising.py

```diff
@@ -74,7 +73,7 @@
 
 print('Learning the dictionary...')
 t0 = time()
-dico = MiniBatchDictionaryLearning(n_components=100, alpha=1, n_iter=500)
+dico = MiniBatchDictionaryLearning(n_components=100, alpha=1, n_iter=500, batch_size=100, n_jobs=4)
```

Comment:
Sorry I left it, I used it for tests. I don't understand why it would break on Windows though, could you explain ?